### PR TITLE
Ui revamp

### PR DESCRIPTION
**Pull request by sebsebmc**
Big update to the home page and some spacing on the search and login pages.

### CODE REVIEW

File: app/models/rating.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@ class Rating < ApplicationRecord
   validates :score, inclusion: { in: [-1, 0, 1] }
   
   SCORE_TO_TEXT = {
-    -1 => '👎',
-    0 => '🤷‍',
-    1 => '👍',
+    -1 => '🙂',
+    0 => '😐',
+    1 => '🙁',
```
Comment (aschmitz-ncc) [inline app/models/rating.rb]: Boo!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -66,4 +65,61 @@ time {
...
+
+
+
+
+
+.homepage {
+  display: flex;
+  flex-direction: column;
+}
+.homepage > * > * {
```
Comment (aschmitz-ncc) [inline app/assets/stylesheets/application.css]: Can we have a slightly less intense CSS query here?

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -66,4 +65,61 @@ time {
...
+  max-width: 960px;
+  margin: auto;
+}
+@media screen and (max-width: 1020px) {
+  .homepage > * > * {
```
Comment (aschmitz-ncc) [inline app/assets/stylesheets/application.css]: Similar to the above.

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (sebsebmc) [reply app/assets/stylesheets/application.css]: Fixed

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (sebsebmc) [reply app/assets/stylesheets/application.css]: Fixed

**Comment by aschmitz-ncc** [comment]
Thanks!

**Pull request merged**